src/main/java/thaumicenergistics/client/gui/GuiEssentiaCellTerminal.java

```diff
@@ -381,7 +381,7 @@ private void updateView() {
             }
 
             // Is the search term in this aspects tag or name?
-            if ((this.searchTerm == "") || (stack.getAspectName().contains(this.searchTerm))
+            if ((this.searchTerm == "") || (stack.getAspectName().toLowerCase().contains(this.searchTerm))
```

Comment:
I'd use `StringUtils.containsIgnoreCase` here to avoid allocating the extra string